File: tests/urls.py
base_url = 'http://localhost:5000'
api_url = f'{base_url}/api/v1'
requests_url = f'{api_url}/requests'


def request_comments_url(id):
    return f'{api_url}/requests/{id}/comments'


def requests_url(id=''):
    if id:
        return f'{api_url}/requests/{id}'
    return f'{api_url}/requests'


def clients_url(id):
    return f'{api_url}/clients/{id}'


def staff_url(id):
    return f'{api_url}/staff/{id}'

File: client/http/urls.js
let baseUrl = 'http://requests2-env.ytcuuufw3p.us-east-2.elasticbeanstalk.com';

if (process.env.NODE_ENV && process.env.NODE_ENV === 'development') {
  baseUrl = 'http://localhost:5000/api/v1';
}

const requestsUrl = `${baseUrl}/requests`;
const requestUrl = id => `${baseUrl}/requests/${id}`;
const requestCommentsUrl = id => `${baseUrl}/requests/${id}/comments`;
const clientsUrl = id => `${baseUrl}/clients/${id}`;
const staffUrl = id => `${baseUrl}/staff/${id}`;

export {
  requestsUrl,
  requestUrl,
  requestCommentsUrl,
  clientsUrl,
  staffUrl,
};
